src/containers/Secrets/Secrets.js

```diff
@@ -99,6 +111,29 @@ export /* istanbul ignore next */ class Secrets extends Component {
             lowContrast
           />
         )}
+        {createSuccess &&
+          (!deleteSuccess && (
+            <InlineNotification
+              kind="success"
+              title="Success:"
```

Comment:
we need to ensure all strings are externalised so they can be translated, use either `intl.formatMessage` or the `FormattedMessage` component as needed.

Once that's done `npm run i18n:extract` will ensure the message bundles are updated correctly with any new strings. These changes should also be included in the PR.

Reply:
Ok, I have changed all of the user-visible strings to use ```intl.formatMessage```